tests/encumbrance_test.cpp

```diff
@@ -132,3 +152,373 @@ TEST_CASE( "tiny_character", "[encumbrance]" )
         test_encumbrance_items( { i }, "torso", longshirt_e, add_trait( "SMALL2" ) );
     }
 }
+
+int throw_cost( const player &c, const item &to_throw );
+int trap_base_detection_score( const Character &p );
```

Comment:
`throw_cost` was not exported in a header, but was used in `throwing_test.cpp`. I assume declaration like this is the preferred way because build times or something?